scss/partials/_blog.scss

```diff
@@ -0,0 +1,404 @@
+        width: 100%;
+    }
+
+    &__title {
+        @include breakpoint(desktop) {
+            font-size: 8.4rem;
+        }
+
+        font-size: 4.2rem;
+    }
+
+    &__blog-author {
+        align-items: center;
+        display: grid;
+        grid-template-columns: 9rem 1fr;
+        padding-bottom: 7.5rem;
+        padding-top: 3.5rem;
+
+        &-img-container {
+            padding-right: 2rem;
+
+            &__author-image {
+                background: none no-repeat center center / cover;
+                width: 6rem;
+                height: 6rem;
+                border-radius: 50%;
+            }
+        }
+
+        &-info {
+            display: inline-block;
+
+            &-title {
+                color: $secondary-green-2;
+                font-size: 1.6rem;
+            }
+
+            &-name {
+                @extend .tk-degular-text;
+                font-size: 2rem;
+            }
+        }
+    }
+
+    &__time-social {
+        border-top: 0.1rem solid $cool-neutral-2;
+        display: grid;
+        grid-template-columns: 18rem 1fr;
+
+        &-read-time {
+            border-right: 0.1rem solid $cool-neutral-2;
+            padding-top: 1.7rem;
+
+            &--p-text,
+            &--p-time {
+                display: inline-block;
+                font-size: 1.8rem;
+            }
+
+            &--p-time {
+                @extend .tk-degular-text;
+                color: $secondary-green-2;
+                padding-right: 3.6rem;
+            }
+        }
+
+        &-links {
+            margin-left: auto;
+            margin-right: 0;
+            padding-top: 2.1rem;
+
+
+            .social-icon__link {
+                margin-right: 1.7rem;
+
+                &:last-child {
+                    margin-right: 0;
+                }
+
+                img {
+                    background-color: $secondary-green-2;
+                }
+
+            }
+        }
+
+    }
+}
+
+// Blog Article Post
+.blog-article {
+    background: rgba(5, 28, 44, 1);
+    background: linear-gradient(180deg, rgba(5, 28, 44, 1) 24.6rem, rgba(255, 255, 255, 1) 24.6rem);
+    padding-top: 3rem;
+    padding-bottom: 0;
+
+
+    &__image {
+        padding-top: 3.8rem;
+        padding-bottom: 6.6rem;
+
+        &-featured {
+            padding-bottom: 13.8rem;
+        }
+    }
+
+    &__post {
+        @include breakpoint(desktop) {
+            margin: 0 19.5rem;
+        }
+
+
+        h2 {
+            @extend .h4;
+        }
+
+        h2,
+        h3,
+        h4 {
+            padding-bottom: 2.7rem;
+        }
+
+        p {
+            color: $cool-neutral-1;
+
+            .dropcap {
+                @extend .tk-degular;
+                color: $accessible-green;
+                float: left;
+                font-size: 16rem;
+                font-weight: 900;
+                line-height: 0.35;
+                margin: 0.1em 0.1em 0.18em 0;
+            }
+        }
+
+        &-blockquote {
```

Comment:
Like you did above, I'd like to see us break BEM here for content simplicity and just have this apply to any default blockquote in a blog post (we can deal with variants later if they come up). At least, as best we can, knowing there's an extra line of text that can't be semantically handled by the `<cite>` element. That will just have to be classed if someone wants it.

Reply:
Done.